config/views.yaml

```diff
@@ -3,8 +3,8 @@ component_readiness:
   - name: 4.18-main
     base_release:
       release: "4.16"
```

Comment:
Should we move this to 4.17 yet?

Reply:
I think the intentional regressions are still being worked out, so a comparison to 4.17 relative to now could well make some things disappear that shouldn't like the console operator and kubeapi error budget burn. Definitely at ga time, but we could also do it once the intentional regressions are all merged.